testing/rstcheck/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Keith Maxwell <keith.maxwell@gmail.com>
+# Maintainer: Keith Maxwell <keith.maxwell@gmail.com>
+pkgname=rstcheck
+_pyname=rstcheck
+pkgver=3.3.1
+pkgrel=0
+pkgdesc="Checks syntax of reStructuredText and code blocks nested within it"
+url="https://github.com/myint/rstcheck"
+arch="noarch"
+license="MIT"
+depends="python3 py3-docutils py3-setuptools"
```

Comment:
```suggestion
depends="python3 py3-docutils"
makedepends="py3-setuptools"
```

Reply:
setuptools is required at runtime, so `py3-setuptools` is deliberately set as a
runtime dependency.

PEP453 explains that it is just an implementation detail that setuptools is
installed by default on Python 3 so the dependency is required.

> other projects which explicitly require setuptools must still provide an
> appropriate dependency declaration

https://www.python.org/dev/peps/pep-0453/#automatic-installation-of-setuptools

`/usr/bin/rstcheck` contains the line `from pkg_resources import
load_entry_point`. `pkg_resources` comes from
`/usr/lib/python3.7/site-packages/pkg_resources/__init__.py` a part of the
py3-setuptools package.

Installing and then removing `py3-setuptools` breaks the version of setuptools
that is bundled with `python3` and if this runtime dependency is not explicit,
breaks this package.

mps mentioned on IRC that this maybe be discouraged in packaging, but a search
of the documentation yields no results.

Reply:
Thanks for taking the time explaining it.